pkgs/tools/networking/ndjbdns/default.nix

```diff
@@ -1,25 +1,30 @@
-{ stdenv, fetchurl, systemd, pkgconfig }:
+{ stdenv, fetchFromGitHub, autoreconfHook, systemd, pkgconfig }:
+
+with stdenv.lib;
 
 stdenv.mkDerivation rec {
   version = "1.06";
```

Comment:
Shouldn't the version be changed? Is this a development version? The repo on GitHub unfortunately doesn't have tags.